### ISSUE TITLE
Support loading of an external image file to the screen res DMD area

### ISSUE THREAD

**Issue by punter1** [vpinball/b2s-backglass#73]
I know the screen res file supports displaying a background frame behind the Backglass via the last line of the screen res file, i.e 
line 17: C:\path\Frame (The path to the location where you have the frame image)

Is it possible to add a line 18 where an image file could be specified that would be displayed as a background frame behind the defined B2S DMD area?

For B2S Backglass that only supports the backglass displya, this would allow user to display a DMD image in the unused DMD display.  

1. Example 1: Display a Bally logo image in the DMD display for an EM table which has no DMD
2. Example 2: Display a FullDMD frame image in the DMD display for a ROM table where vPinMame DMD would sit on top of the FullDMD image

I did see Feature Request #50 but not sure if that can be used to accomplish the above.

Thanks,
gtxjoe

**Comment by JockeJarre** [comment]
I do not have any full DMD display on my cab, but I would say that the #50 should do what you want.
Check out [B2SScreenResIdentifierurl](https://github.com/vpinball/b2s-backglass/wiki/B2SScreenResIdentifierurl) to move the background where you want it and your full DMD display should be possible here.

**Comment by gtxjoe** [comment]
Thanks Karl. Agreed.  Thanks for hint on freezy DMDext overlays, I will try this approach when I have time

**Comment by JockeJarre** [comment]
But I am still sure you can move the "background" wherever you want. For example I use it _sometimes_ to add a grill to pup tables. 
See [here](https://github.com/vpinball/b2s-backglass/wiki/Dual-Screen-examples). The grill is just defined as a background in the res file:

```Python
# V2.0.0-c311a21
# File is saved with B2S_ScreenResIdentifier release 2.0.0 adding the first line means it will not flip the Backglass & Background size and position
# Playfield Screen resolution width/height
1920
1080
# width/height of the Backglass
1280
655
# Define Backglass using the screen index (=x) -> **It is always the second screen from left**
=2
# Backglass x/y position relative to the upper left corner Of the screen selected
0
0
# width/height Of the B2S (or Full) DMD area In pixels
676
320
# X/Y position Of the DMD area relative To the upper left corner of the backglass screen
303
681
# Y-flip, flips the LED display upside down
0
# X/Y position of the background
0
655
# width/height of the background
1280
369
# path to the background image (C:\path\Frame)
C:\Visual Pinball\B2SServer\Backgrounds\SpikeTypeGrill.png
```

**Comment by Karl030167** [comment]
But what if...
Still want to use the "line 17" for showing left/right borders of a 4:3 b2s on 16:9 screen
&
Also want to use "new feature line 18" for showing a picture (...Bally logo, ...) at separate DMD screen ?

Maybe a wish for Christmas 😇

**Comment by JockeJarre** [comment]
there will not be any YALIAOCF (Yet Another Line In a Over Complicated File) coming from me at least.
If we should add something like this YASTPH (Yet Another Show This Picture Here), we need to make use of a better file format, like I describe in another issue here https://github.com/vpinball/b2s-backglass/issues/23

**Comment by Karl030167** [comment]
Yes, you're right. This is no life-saving request. We already CAN do this kind of visualisation. With adding a DMD picture into .directb2s with b2s designer, or another option is to use the frontend to display a picture at DMD while playing.
Your way to implement the background picture as border to 4:3 b2s is great, indeed.
And a complete new way to config b2s with XML would be very fine...
 (as long as very old exe b2s backglasses - used by some very beautiful vp9 tables - still working... They only works with old plain screenres.txt)

So maybe you can "close and forget" gtxjoe's and my idea of a line 18. And maybe there will be such kind of feature in future with b2sserver XML setting 😀

**Comment by JockeJarre** [comment]
Closing as a duplicate to [#23 ](https://github.com/vpinball/b2s-backglass/issues/23) 
When it is solved, using a better file format for the ScreenRes files, I guess it should be possible to add as many pictures you want?
